src/utils/valueHelpers.ts

```diff
@@ -10,10 +10,8 @@ export function getStringForValue(
       return quotedStrings ? `"${value}"` : `${value}`;
     case SchemaType.Number:
     case SchemaType.Integer:
-      return value.toString();
     case SchemaType.Boolean:
-      return value.toString();
     default:
-      throw new Error(`Unexpected value type: ${valueType.type}`);
+      return value.toString();
```

Comment:
Good point. This default used as a fallback, but it may be risky to have it in place, for example, if there is a type that needs special handling we'd just do a toString and won't fail but will generate an invalid output.

I'm going back to throwing so we are explicit on what we're handling